test/integration/server-selection/server_selection.prose.operation_count.test.ts

```diff
@@ -147,13 +147,19 @@ describe('operationCount-based Selection Within Latency Window - Prose Test', fu
   it('equally distributes operations with both hosts are fine', TEST_METADATA, async function () {
     const collection = client.db('test-db').collection('collection0');
 
+    const numberTaskGroups = 10;
+    const numberOfTasks = 1000;
+    const totalNumberOfTasks = numberTaskGroups * numberOfTasks;
+
     // Step 8: Start 10 concurrent threads / tasks that each run 100 findOne operations with empty filters using that client.
-    await Promise.all(Array.from({ length: 10 }, () => runTaskGroup(collection, 100)));
+    await Promise.all(
```

Comment:
Just a suggestion to add a comment explaining why we're not actually doing Step 8 as written